src/platform/nrfconnect/CHIPDevicePlatformConfig.h

```diff
@@ -215,11 +215,7 @@
 #endif // CHIP_DEVICE_CONFIG_CHIP_TASK_PRIORITY
 
 #ifndef CHIP_DEVICE_CONFIG_CHIP_TASK_STACK_SIZE
-#ifdef CONFIG_LTO
```

Comment:
Why this change? It's not mentioned in the PR description.

Reply:
We need to increase the stack size to 8k anyway, so it doesn't matter whether it is the LTO build or not.

Reply:
But... It is regarding CC + Oberon only, so I will add the ifdef there to not impact on the memory requirements table,